### ISSUE TITLE
slider: missing ARIA attributes for the helperText

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#3665]
these are missing on the appropriate element:

```
aria-describedby={this.helperText ? this.helperTextId : undefined}
aria-controls={this.helperTextId}
```

See implementation in `limel-input-field` for further info
